pkg/controller/kubelet-config/kubelet_config_nodes_test.go

```diff
@@ -218,8 +222,13 @@ func TestNodeConfigCustom(t *testing.T) {
 
 			mcList, err = c.client.MachineconfigurationV1().MachineConfigs().List(context.TODO(), metav1.ListOptions{})
 			require.NoError(t, err)
+			// Now expecting 1 MachineConfig:
+			// 1. Node config MC for worker (97-worker-generated-kubelet)
+			// Note: The pre-existing nodeKeyCustom (97-custom-generated-kubelet) is cleaned up by
+			// cleanUpDuplicatedMC because it was created without the proper annotation
 			require.Len(t, mcList.Items, 1)
-			require.NotEqual(t, nodeKeyCustom, mcList.Items[0].Name)
+			// Verify MachineConfig is present
+			require.Equal(t, "97-worker-generated-kubelet", mcList.Items[0].Name)
```

Comment:
Why is this change necessary?
`nodeKeyCustom` is already derived/generated from the `getManagedNodeConfigKey` method.
Better not to have a hard-coded value here.